deployment/cdk/opensearch-service-migration/lib/stack-composer.ts

```diff
@@ -207,7 +209,6 @@ export class StackComposer {
         const captureProxyExtraArgs = this.getContextForType('captureProxyExtraArgs', 'string', defaultValues, contextJSON)
         const elasticsearchServiceEnabled = this.getContextForType('elasticsearchServiceEnabled', 'boolean', defaultValues, contextJSON)
         const kafkaBrokerServiceEnabled = this.getContextForType('kafkaBrokerServiceEnabled', 'boolean', defaultValues, contextJSON)
-        const targetClusterEndpoint = this.getContextForType('targetClusterEndpoint', 'string', defaultValues, contextJSON)
```

Comment:
Removing a parameter is a major change, let's make sure we get input from the whole team on this as it will break all existing cdk.context.json that we/customers have which specify targetClusterEndpoint. 

We should think about if we want to support both fields for a time period.

Another approach could be to add "targetAuthConfig" as a separate key from the endpoint and not change this key, which I'm in favor of as I see it as less intrusive.

If we move forward with removing this, let's make sure to scrub it from all our wiki's and documentation, i see a reference to this in the cdk README remaining.

Reply:
Okay, yeah, I like that idea. I'll rework this to support `targetClusterEndpoint` and `targetClusterAuthConfig`